buildSrc/src/main/groovy/nva.nvi.dynamodblocal.gradle

```diff
@@ -8,6 +8,7 @@ repositories {
 }
 
 dependencies {
+//    https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.
```

Comment:
Remove commented code?

Reply:
It's not code. It's a link to documentation concerning updating to newer version